surfify/utils/io.py

```diff
@@ -121,3 +122,22 @@ def write_freesurfer(vertices, triangles, surf_file):
     nibabel.freesurfer.io.write_geometry(surf_file, vertices, triangles,
                                          create_stamp="",
                                          volume_info=None)
+
+
+class HidePrints:
+    def __init__(self, hide_all=False):
```

Comment:
this is a generic function that can be removed from the module.
note that if the logging module is used in combination with the setup_logging function it must be enough to remove prints.